BonovoBluetooth/src/com/bonovo/bluetooth/BonovoBluetoothHandfree.java

```diff
@@ -359,49 +387,59 @@ protected void onResume() {
 			}
 		}
 		mDialpadPressCount = 0;
+		

```

Comment:
Can we try not to make so many whitespace changes like these?  In the future if there is another source leak it just makes it a bit harder to do a comparison of changes.